### PR TITLE
HTML-CSS-Popup

### PR DESCRIPTION
**Pull request by MosQuitO404**
### HTML-CSS-Popup

[Demo](https://mosquito404.github.io/popup_html_css/) |
[Code base](https://github.com/MosQuitO404/popup_html_css)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by MosQuitO404** [comment]
> Hey!
> 
> Congratulations with your PR! 😎😎😎
> 
> Please, be sure you haven't followed [common mistakes](https://kottans.org/documentation/docs/doc/code-review/#popup-task).
> 
> Also, be aware, that if you would silently ignore this recommendation, a mentor may think that you are still working on fixes. And your PR will not be reviewed. 😒

@A-Ostrovnyy @OleksiyRudenko Typical mistakes were fixed. Review, please.

File: submissions/MosQuitO404/HTML-CSS-Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                    <a class="nav-items-element" href="#">Blog</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Integrations</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="https://github.com/MosQuitO404" target="_blank">Contacts</a>
+                </li>
+            </ul>
+            <div id="popup-section">
```
Comment (A-Ostrovnyy) [inline submissions/MosQuitO404/HTML-CSS-Popup/index.html:38]: When a user opens the popup focus must go to the first interactive element, usually, it's a close button, but in our case when a user opens the popup and presses the` Tab` key focus must go to the first link inside the popup. In your case order of focused elements is broken. :cry:

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome design!

File: submissions/MosQuitO404/HTML-CSS-Popup/index.html (hunk 1 above)
Comment (MosQuitO404) [reply submissions/MosQuitO404/HTML-CSS-Popup/index.html:28]: Hi! Bugs were fixed and added adaptive design. Review pls.

File: submissions/MosQuitO404/HTML-CSS-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Home</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">About</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Pricing</a>
+                </li>
+            </ul>
```
Comment (A-Ostrovnyy) [inline submissions/MosQuitO404/HTML-CSS-Popup/index.html:25]: All these elements must be a part of the one `ul` element.
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One last issue)

File: submissions/MosQuitO404/HTML-CSS-Popup/index.html (hunk 2 above)
Comment (MosQuitO404) [reply submissions/MosQuitO404/HTML-CSS-Popup/index.html]: Hi! Bug were fixed. Review pls.

File: submissions/MosQuitO404/HTML-CSS-Popup/style.css [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+  opacity: 100%;
+}
+
+.popup-set-li:hover {
+  opacity: 100%;
+  box-shadow: 0px 3px 14px rgba(74, 58, 255, 0.03), 0px -2px 4px rgba(20, 20, 43, 0.02), 0px 12px 24px rgba(20, 20, 43, 0.04);
+  border-radius: 10px;
+}
+
+.popup-set-li:hover > span > .popup-element > .icons-pictures {
```
Comment (A-Ostrovnyy) [inline submissions/MosQuitO404/HTML-CSS-Popup/style.css:245]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid too specific CSS selectors. [Read more about it](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

File: submissions/MosQuitO404/HTML-CSS-Popup/style.css [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+  #integrations-button {
+    order: 5;
+  }
+  #contacts-button {
+    order: 6;
+  }
+  #popup-wrapper {
+    order: 7;
+    flex-basis: 100%;
+  }
```
Comment (A-Ostrovnyy) [inline submissions/MosQuitO404/HTML-CSS-Popup/style.css:62]: No need to set order for all elements? you can just set `order: 1` to icon item to move him to the last position in list

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Goor progress!

File: submissions/MosQuitO404/HTML-CSS-Popup/style.css (hunk 4 above)
Comment (MosQuitO404) [reply submissions/MosQuitO404/HTML-CSS-Popup/style.css]: Done!

File: submissions/MosQuitO404/HTML-CSS-Popup/style.css (hunk 3 above)
Comment (MosQuitO404) [reply submissions/MosQuitO404/HTML-CSS-Popup/style.css]: Ок, done!

File: submissions/MosQuitO404/HTML-CSS-Popup/index.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+                        <div class="popup-set">
+                            <ul class="popup-set-ul">
+                                <li class="popup-set-li">
+                                    <span>
+                                        <a class="popup-element" href="#"><img src="sources/icons/free-icon-house-3845763.png" class="icons-pictures" alt=""><br>Home</a>
+                                    </span>
+                                </li>
+                                <li class="popup-set-li">
+                                    <span>
+                                        <a class="popup-element" href="#"><img src="sources/icons/free-icon-contacts-3845847.png" class="icons-pictures" alt=""><br>Contacts</a>
```
Comment (A-Ostrovnyy) [inline submissions/MosQuitO404/HTML-CSS-Popup/index.html:38]: In the future avoid using `<br>` for defining block in new line)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

File: submissions/MosQuitO404/HTML-CSS-Popup/index.html (hunk 5 above)
Comment (MosQuitO404) [reply submissions/MosQuitO404/HTML-CSS-Popup/index.html:38]: Ok, thanks 🙏☺️ I understand 😌

**Pull request merged**